drivers/entropy/entropy_nrf5.c

```diff
@@ -16,6 +16,7 @@ struct entropy_nrf5_dev_data {
 #define DEV_DATA(dev) \
 	((struct entropy_nrf5_dev_data *)(dev)->driver_data)
 
+#ifndef CONFIG_BT
```

Comment:
#if !defined(CONFIG_BT)

Also, add your copyright if you wish to this file too.

Reply:
done